tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -95,7 +95,9 @@ export class WebGPUBackend extends KernelBackend {
   shaderc: shaderc.Shaderc;
   compiler: shaderc.Compiler;
   compileOpts: shaderc.CompileOptions;
-  commandQueue: GPUCommandEncoder[];
+  commandQueue: GPUCommandBuffer[];
```

Comment:
Could you add brief (1-2 sentence) annotations describing what commandEncoder / passEncoder are and how they're different?